lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -406,11 +406,11 @@ WasmBinaryReader::EndOfModule()
 void
 WasmBinaryReader::ReadMemorySection()
 {
-    // TODO: change to use multiple of page size
-    uint32 minSize = 1 << ReadConst<UINT8>();
-    uint32 maxSize = 1 << ReadConst<UINT8>();
+    UINT length = 0;
+    UINT minPage = LEB128(length);
+    UINT maxPage = LEB128(length);
```

Comment:
Yeah the section length validation (and other validation items) will have to be done in a future change. 
For the time being we need to have functionality working and we assume the binaries are valid. 
Obviously we won't ship it like this though